canonical-web-and-design/snapcraft.io
# Revision attributes icons

Fixes #2226 
Add Lauchpad icon to revisions built on LP
Adds Lauchpad revisions to 'Available revisions' dropdown

### QA
- ./run or demo
- go to releases page of any snap built on lauchpad
- check if revisions built on LP have icon in releases table
- check if revisions built on LP have icon in releases table tooltip
- check if revisions built on LP have icon in revision history build column
- if all revisions in the channel are from the same build request icon should be shown under channel name as well
- click on available revisions dropdown, Launchpad revisions should be listed there

<img width="1295" alt="Screenshot 2019-09-05 at 09 53 04" src="https://user-images.githubusercontent.com/83575/64322709-631e1900-cfc3-11e9-92be-fca90141ca4d.png">
